### PR TITLE
Add property name validation

### PR DESCRIPTION
**Pull request by danangmassandy**
This is for:

- https://github.com/kartoza/sawps/issues/1340
- https://github.com/kartoza/sawps/issues/1286

Preview error message:
image

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 47 sec

File: django_project/frontend/src/containers/MainPage/Upload/Step1.tsx [hunk 1]
```diff
@@ -52,34 +53,54 @@ export default function Step1(props: Step1Interface) {
...
+            response => {
+                let _isAvailable = response.data['available']
+                if (_isAvailable) {
+                    if (property.id) {
+                        // if it has property.id, then this is to update property
+                        let _data:PropertyInterface = {
+                            ...property
+                        }
+                        setSavingProperty(true)
+                        postData(`${PROPERTY_UPDATE_INFORMATION_URL}`, _data).then(
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Upload/Step1.tsx:73]: @danangmassandy could we extract this into a new function instead?

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/src/containers/MainPage/Upload/Step1.tsx (hunk 1 above)
Comment (danangmassandy) [reply django_project/frontend/src/containers/MainPage/Upload/Step1.tsx]: updated Dimas,,

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 53 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 51 sec

**Pull request merged**